openmc/settings.py

```diff
@@ -444,15 +487,42 @@ def source_file(self, source_file):
 
         self._source_file = source_file
 
-
     def set_source_space(self, stype, params):
+        """Defined the spatial bounds of the external/starting source.
+
+        Parameters
+        ----------
+        stype : str
+            The type of spatial distribution. Valid options are "box" and
+            "point". A "box" spatial distribution has coordinates sampled
+            uniformly in a parallelepiped. A "point" spatial distribution has
+            coordinates specified by a triplet.
+        params : tuple or list or ndarray
+            For a "box" spatial distribution, ``params`` should be given as six
+            real numbers, the first three of which specify the lower-left corner
+            of a parallelepiped and the last three of which specify the
+            upper-right corner. Source sites are sampled uniformly through that
+            parallelepiped.
+
+            To filter a "box" spatial distribution by fissionable material,
```

Comment:
I think that the description for how to use the "fission" tag is confusing both here and in the online [documentation](http://mit-crpg.github.io/openmc/usersguide/input.html#source-element). If "fission" is an acceptable "type," why is it not included under "stype"?